### PR TITLE
[feat] Enhancements for modals

### PR DESCRIPTION
**Pull request by gregorylegarec**

### CODE REVIEW

File: react/Modal/index.jsx [hunk 1]
```diff
@@ -6,26 +6,32 @@ import styles from './index.styl'
...
         </button>
-        {description}
+        { description && <div className={classNames(styles['coz-modal-content'], styles['coz-description'])}>
+            {description}
+          </div>
+        }
+        { children }
         <div className={styles['coz-modal-buttons']}>
-          <button className={styles[cancelType]} onClick={cancelAction}>
+          { (cancelText || validateAction) && <button className={styles[cancelType]} onClick={cancelAction}>
```
Comment (y-lohse) [inline react/Modal/index.jsx]: Did you mean `cancelText || cancelAction` here?

**Comment by gregorylegarec** [comment]
@GoOz

File: stylus/ui-components/modals.styl [hunk 2]
```diff
@@ -36,22 +36,33 @@ $modals
...
+            padding 0 1.5em
+
+        .coz-modal-section
+            border-top 1px solid grey-09
+
         .coz-modal-title
-            right-spaced(1.8em)
-            bottom-spaced(1.8em)
             font-weight bold
+            margin .33em 0 1em
```
Comment (GoOz) [inline stylus/ui-components/modals.styl]: Wrong alignment here :)

File: stylus/ui-components/modals.styl [hunk 3]
```diff
@@ -36,22 +36,33 @@ $modals
...
 
-        .coz-modal-close
+        .coz-btn-modal-close
             position        absolute
-            top             2em
-            right           1.8em
-            background-size 22px
+            top             1.3em
+            right           1em
+            margin  0
```
Comment (GoOz) [inline stylus/ui-components/modals.styl]: wrong alignement here too

File: stylus/ui-components/modals.styl (hunk 2 above)
Comment (gregorylegarec) [inline stylus/ui-components/modals.styl]: 🎵  It makes me craaaaaazyyyyy 🎵

**Comment by GoOz** [comment]
👍

**Pull request merged**